### PR TITLE
done nav-bar,sidebar

### PR DESCRIPTION
**Pull request by srishtigautam21**
Added nav-bar and side bar to my CL and did changes in the file structure

### CODE REVIEW

File: css/components.css [hunk 1]
```diff
@@ -5,15 +5,19 @@
...
 
 :root{
-    --primary-color:#22c55e;
+    /* --primary-color:#22c55e; */
+    --primary-color:#16a34a;
     --secondary-color:#bbf7d0;
-    --primary-hover:#16a34a;
+    /* --primary-hover:#16a34a; */
+    --social-icon-color:#475569;
+    --primary-hover:#15803d;
```
Comment (zoef-yt) [inline css/components.css:16]: suggestion: just a suggestion you should give your color name as --color-primary.

**Review by zoef-yt — APPROVED** [review]
Looks good for merge.

File: css/components.css (hunk 1 above)
Comment (srishtigautam21) [reply css/components.css:16]: ok i will make these changes

File: component/Alert/alert.css [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+/* .main-container{
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
-}
+} */
 .alert{
     color:var(--white-color);
     padding: 1.5rem 0rem 1.5rem 1.5rem;
```
Comment (itsmebipul00) [inline component/Alert/alert.css:9]: you can remove the unit while using 0

**Comment by itsmebipul00** [comment]
Good for merge!

File: component/Alert/alert.css (hunk 2 above)
Comment (srishtigautam21) [reply component/Alert/alert.css:9]: ok didn't knew that will keep in mind

**Pull request merged**